src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -171,6 +171,22 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructorReminders(
     }
     
     /**
+     * Generates the feedback session closed emails for the given {@code session}.
+     */
+    public List<EmailWrapper> generateFeedbackSessionClosedEmails(FeedbackSessionAttributes session) {
+
+        boolean isEmailNeeded = fsLogic.isFeedbackSessionForStudentsToAnswer(session);
```

Comment:
@tarun14110 in line with this decision, we need to include the students as well.